### ISSUE TITLE
SurfaceTool.commit flags argument is not documented

### ISSUE THREAD

**Issue by Zylann** [godotengine/godot-docs#2694]
`SurfaceTool` has this function:
```cpp
Ref<ArrayMesh> commit(const Ref<ArrayMesh> &p_existing = Ref<ArrayMesh>(), uint32_t p_flags = Mesh::ARRAY_COMPRESS_DEFAULT);
```

But the last argument looks like this in the doc:
```
int flags=97280 )
```
And it isn't explained what this value represent either in the description.

We should explain it comes from `Mesh.ARRAY_FORMAT_*` and that the default value is `Mesh.ARRAY_COMPRESS_DEFAULT`.

**Comment by Nutriz** [comment]
Hi,

I will fix that.

Do you know if the flag can be any value of `Mesh.ARRAY_*` ? Or only  `Mesh.ARRAY_COMPRESS_*` values ?

**Comment by clayjohn** [comment]
@Nutriz You can use Mesh.ARRAY_FORMAT_* values or the Mesh.ARRAY_FLAG_* values, but they will be ignored.

**Comment by Nutriz** [comment]
Ok, so I will only mention `ARRAY_COMPRESS_*` in the doc.

**Comment by akien-mga** [comment]
Fixed by godotengine/godot#32460.
